org_status/status_providers/__init__.py

```diff
@@ -38,3 +38,7 @@ def status_from_badge_svg(self, svg):
             for variant in variants:
                 if variant in svg:
                     return status
+
+    @classmethod
+    def get_status_provider_status():
```

Comment:
This method should take at least take one parameter as it is a class method. Here is an interesting article https://stackabuse.com/pythons-classmethod-and-staticmethod-explained.